src/otx/algo/detection/mmconfigs/ssd_mobilenetv2.yaml

```diff
@@ -50,8 +50,6 @@ bbox_head:
     - 96
     - 320
   use_depthwise: true
-  norm_cfg:
```

Comment:
It's just a qeustion. Why is only this removed?

Reply:
All of conv_cfg, act_cfg things should be removed, but currently we use mmdeploy, and mmdeply build model by itself. So we must keep some values for export procedure. When we export using native exporter, then all other elements could be removed